roles/openshift_health_checker/openshift_checks/logging/fluentd.py

```diff
@@ -0,0 +1,174 @@
+# pylint: disable=missing-docstring
+"""
+Module for performing checks on an Fluentd logging deployment
+"""
+
+import json
+
+from openshift_checks import get_var
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class Fluentd(LoggingCheck):
+    """Module that checks an Fluentd deployment shipped with OCP"""
+
+    name = "fluentd"
+    tags = ["health", "logging"]
+
+    logging_namespace = None
+
+    def run(self, tmp, task_vars):
+        """Check various things and gather errors. Returns: result as hash"""
```

Comment:
Do you have a quick pointer to that? I assume this is a case of "EFK got left behind in an upgrade" if it's not there...